### ISSUE TITLE
Add onnxruntime_extensions in onnxruntime docker file

### ISSUE THREAD

**Issue by Sriharan-Manogaran** [microsoft/onnxruntime#8639]
**Is your feature request related to a problem? Please describe.**
I was using Universal Sentence Encoder Multilingual using onnxruntime and onnxruntime_extensions python SDKs which is working fine. But when I moved to [onnxruntime docker image](https://github.com/microsoft/onnxruntime/blob/master/dockerfiles/Dockerfile.server) and received the error for `SentencepieceTokenizer`. So can we have this part of our docker file and onnxruntime session?

Similarly, TF requires `tensorflow-text` in SDK but in a docker image, it was supported by default.

**System information**
- ONNX Runtime version (you are using): local_build ([as specified in this file](https://github.com/microsoft/onnxruntime/blob/master/dockerfiles/Dockerfile.server))

**Describe the solution you'd like**
We can include the `onnxruntime_extensions` module in the docker file and in the ort session.

**Comment by wenbingl** [comment]
check this dockerfile https://github.com/microsoft/onnxruntime/pull/8690 to see if it meets your requirements?

**Comment by Sriharan-Manogaran** [comment]
@wenbingl 
This is good, but this extension should be available in all DockerFile as `onnxruntime` library exists. At least in CPU, GPU, and server DockerFiles, we should have it to provide support to NLP-based models.

**Comment by wenbingl** [comment]
Agree, but updating CPU/GPU/TensorRT and others needs an amount of time, let's put it on the backlog.
The dockerfile in PR provides an example of how to build them together, if anyone needs it immediately, they can make the change accordingly.

**Comment by Sriharan-Manogaran** [comment]
Sure, that's perfect.
Kindly add this to the backlog or let me know how to do it.

**Comment by wenbingl** [comment]
@snnn , can we just merge this extension build step into CPU docker? GPU and others could be done later.

**Comment by snnn** [comment]
I don't know who is using these docker.

**Comment by faxu** [comment]
ONNX Runtime Server [is deprecated](https://github.com/microsoft/onnxruntime/blob/master/docs/ONNX_Runtime_Server_Usage.md). Because onnxurntime-extensions is still experimental at this time, we'd like to keep it out of the CPU/GPU dockerfiles in main onnxruntime. That said, if you have feedback on onnxruntime-extensions, we'd love to hear to help us understand usability and future additions to consider moving it out of the experimental stage. You can send me an email at:
![image](https://user-images.githubusercontent.com/20780999/129270120-4776e469-3358-4618-bd01-aa17e1f3e5a0.png)
